examples/twittermap/web/conf/application.conf

```diff
@@ -77,8 +77,9 @@ bounded-mailbox {
 
```

Comment:
I think this file should not be updated for this PR.

Reply:
Yep, we have fixed this I think.

Reply:
Let's not change the application.conf file.